src/components/Header.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <header>
+        <img src="../assets/postit.png">La Postiterie
+    </header>
+</template>
+
+<script>
+export default {
+    name: 'Header'
+}
+</script>
+
+<style scoped>
+
+header {
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    font-size : 1.8em;
+    text-align:center;
+    font-weight: 900;
+    background-color: #478BF9;
+    padding-right: 12px;
```

Comment:
These 4 lines can be one: `padding: 4px 12p 5px 0;`

Plus, by convention, we always order `xxx-side` properties this way: top, right, bottom, left (clockwise starting by top)